content/en/docs/components/training/overview.md

```diff
@@ -0,0 +1,99 @@
++++
+title = "Overview"
+description = "An overview of Kubeflow Training Operator"
+weight = 5
++++
+
+Kubeflow Training Operator is a Kubernetes-native project for fine-tuning and scalable
+distributed training of machine learning (ML) models created with various ML frameworks such as
+PyTorch, Tensorflow, XGBoost, and others.
+
+Training Operator allows you to use Kubernetes workloads to effectively train your large models
+via Kubernetes Custom Resources APIs or using Training Operator Python SDK.
+
+Training Operator implements centralized Kubernetes controller to orchestrate distributed training jobs.
+
+## Architecture
+
+This diagram shows the major features of Training Operator and supported ML frameworks.
+
+<img src="/docs/components/training/images/training-operator-overview.drawio.png"
+  alt="Training Operator Overview"
+  class="mt-3 mb-3">
+
+## Custom Resources for ML Frameworks
+
+To perform distributed training Training Operator implements the following
+[Custom Resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/):
+
+| ML Framework | Custom Resource |
+| ------------ | --------------- |
+| PyTorch      | PyTorchJob      |
+| Tensorflow   | TFJob           |
+| XGBoost      | XGBoostJob      |
+| MPI          | MPIJob          |
```

Comment:
As I can see `overview.md`, we mention only the training-operator. So it would be nice to mention that the training operator supports only MPIJob v1, and then we can put a link for the mpi-operator (https://github.com/kubeflow/website/blob/f5e097692e24ffe7722bc94d6ee956812f2965bf/content/en/docs/components/training/mpi.md).

Reply:
@tenzen-y @alculquicondor @kuizhiqing  @johnugeorge I added a few info about MPI Operator: https://github.com/kubeflow/website/blob/6db835dd4f290d3bbc97dcbe538dde093005e1b6/content/en/docs/components/training/overview.md#what-is-kubeflow-training-operator-.
Please let me know what do you think.